crash for Moonshae Pixie & Myrkul's Edict

crash with #1048

log:

```
EDT > java.lang.RuntimeException: crash in raw Ability, check card script of Myrkul's Edict
    at forge.game.card.CardFactoryUtil.addAbilityFactoryAbilities(CardFactoryUtil.java:608)
    at forge.game.card.CardFactory.readCardFace(CardFactory.java:450)
    at forge.game.card.CardFactory.readCard(CardFactory.java:361)
    at forge.game.card.CardFactory.getCard(CardFactory.java:246)
    at forge.game.card.CardFactory.getCard(CardFactory.java:242)
    at forge.game.card.Card.fromPaperCard(Card.java:6532)
    at forge.game.card.Card.getCardForUi(Card.java:6543)
    at forge.game.card.CardView.getCardForUi(CardView.java:64)
    at forge.card.CardRenderer.drawCard(CardRenderer.java:603)
    at forge.itemmanager.views.ImageView$ItemInfo.draw(ImageView.java:991)
    at forge.itemmanager.views.ImageView$Pile.draw(ImageView.java:920)
    at forge.itemmanager.views.ImageView$Group.draw(ImageView.java:829)
    at forge.Graphics.draw(Graphics.java:176)
    at forge.toolbox.FContainer.draw(FContainer.java:64)
    at forge.toolbox.FScrollPane.draw(FScrollPane.java:332)
    at forge.Graphics.draw(Graphics.java:176)
    at forge.toolbox.FContainer.draw(FContainer.java:64)
    at forge.Graphics.draw(Graphics.java:176)
    at forge.toolbox.FContainer.draw(FContainer.java:64)
    at forge.Graphics.draw(Graphics.java:176)
    at forge.toolbox.FContainer.draw(FContainer.java:64)
    at forge.screens.FScreen.draw(FScreen.java:164)
    at forge.Forge.render(Forge.java:887)
    at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Window.update(Lwjgl3Window.java:387)
    at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.loop(Lwjgl3Application.java:192)
    at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:166)
    at forge.app.Main.main(Main.java:148)
```

and

```
EDT > java.lang.RuntimeException: Error in Trigger for Card: Moonshae Pixie
    at forge.game.trigger.TriggerHandler.parseTrigger(TriggerHandler.java:141)
    at forge.game.card.CardFactory.readCardFace(CardFactory.java:409)
    at forge.game.card.CardFactory.readCard(CardFactory.java:361)
    at forge.game.card.CardFactory.getCard(CardFactory.java:246)
    at forge.game.card.CardFactory.getCard(CardFactory.java:242)
    at forge.game.card.Card.fromPaperCard(Card.java:6532)
    at forge.game.card.Card.getCardForUi(Card.java:6543)
    at forge.game.card.CardView.getCardForUi(CardView.java:64)
    at forge.card.CardRenderer.drawCard(CardRenderer.java:603)
    at forge.itemmanager.views.ImageView$ItemInfo.draw(ImageView.java:991)
    at forge.itemmanager.views.ImageView$Pile.draw(ImageView.java:920)
    at forge.itemmanager.views.ImageView$Group.draw(ImageView.java:829)
    at forge.Graphics.draw(Graphics.java:176)
    at forge.toolbox.FContainer.draw(FContainer.java:64)
    at forge.toolbox.FScrollPane.draw(FScrollPane.java:332)
    at forge.Graphics.draw(Graphics.java:176)
    at forge.toolbox.FContainer.draw(FContainer.java:64)
    at forge.Graphics.draw(Graphics.java:176)
    at forge.toolbox.FContainer.draw(FContainer.java:64)
    at forge.Graphics.draw(Graphics.java:176)
    at forge.toolbox.FContainer.draw(FContainer.java:64)
    at forge.screens.FScreen.draw(FScreen.java:164)
    at forge.Forge.render(Forge.java:887)
    at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Window.update(Lwjgl3Window.java:387)
    at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.loop(Lwjgl3Application.java:192)
    at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:166)
    at forge.app.Main.main(Main.java:148)
```

just tried to open deck editor, type Myrkul's Edict, after hitting enter, crashes... 
```
EDT > java.lang.RuntimeException: crash in raw Ability, check card script of Myrkul's Edict
	at forge.game.card.CardFactoryUtil.addAbilityFactoryAbilities(CardFactoryUtil.java:608)
	at forge.game.card.CardFactory.readCardFace(CardFactory.java:450)
	at forge.game.card.CardFactory.readCard(CardFactory.java:361)
	at forge.game.card.CardFactory.getCard(CardFactory.java:246)
	at forge.game.card.CardFactory.getCard(CardFactory.java:242)
	at forge.game.card.Card.fromPaperCard(Card.java:6532)
	at forge.game.card.Card.getCardForUi(Card.java:6543)
	at forge.game.card.CardView.getCardForUi(CardView.java:64)
	at forge.card.CardRenderer.drawCard(CardRenderer.java:606)
	at forge.itemmanager.views.ImageView$ItemInfo.draw(ImageView.java:991)
	at forge.itemmanager.views.ImageView$Pile.draw(ImageView.java:920)
	at forge.itemmanager.views.ImageView$Group.draw(ImageView.java:829)
	at forge.Graphics.draw(Graphics.java:176)
	at forge.toolbox.FContainer.draw(FContainer.java:64)
	at forge.toolbox.FScrollPane.draw(FScrollPane.java:332)
	at forge.Graphics.draw(Graphics.java:176)
	at forge.toolbox.FContainer.draw(FContainer.java:64)
	at forge.Graphics.draw(Graphics.java:176)
	at forge.toolbox.FContainer.draw(FContainer.java:64)
	at forge.Graphics.draw(Graphics.java:176)
	at forge.toolbox.FContainer.draw(FContainer.java:64)
	at forge.screens.FScreen.draw(FScreen.java:164)
	at forge.Forge.render(Forge.java:887)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Window.update(Lwjgl3Window.java:387)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.loop(Lwjgl3Application.java:192)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:166)
	at forge.app.Main.main(Main.java:148)
Caused by: java.lang.RuntimeException: AbilityFactory:getAbility: crash when trying to create ability  of card: Myrkul's Edict
	at forge.game.ability.AbilityFactory.getAbility(AbilityFactory.java:157)
	at forge.game.ability.AbilityFactory.getAbility(AbilityFactory.java:116)
	at forge.game.ability.AbilityFactory.getAbility(AbilityFactory.java:113)
	at forge.game.card.CardFactoryUtil.addAbilityFactoryAbilities(CardFactoryUtil.java:595)
	... 26 more
Caused by: java.lang.RuntimeException: AbilityFactory : getAbility -- no API in Myrkul's Edict: ChoosePlayer | Defined$ You | Choices$ Player.Opponent | SubAbility$ DBSac | StackDescription$ SpellDescription | SpellDescription$ 1—9 VERT Choose an opponent. That player sacrifices a creature.
	at forge.game.ability.AbilityFactory.getAbility(AbilityFactory.java:145)
	at forge.game.ability.AbilityFactory.getAbility(AbilityFactory.java:174)
	at forge.game.ability.AbilityFactory.getSubAbility(AbilityFactory.java:457)
	at forge.game.ability.AbilityFactory.getAbility(AbilityFactory.java:299)
	at forge.game.ability.AbilityFactory.getAbility(AbilityFactory.java:179)
	at forge.game.ability.AbilityFactory.getAbility(AbilityFactory.java:148)
	... 29 more
```

> just tried to open deck editor, type Myrkul's Edict, after hitting enter, crashes...
> 
> ```
> EDT > java.lang.RuntimeException: crash in raw Ability, check card script of Myrkul's Edict
> 	at forge.game.card.CardFactoryUtil.addAbilityFactoryAbilities(CardFactoryUtil.java:608)
> 	at forge.game.card.CardFactory.readCardFace(CardFactory.java:450)
> 	at forge.game.card.CardFactory.readCard(CardFactory.java:361)
> 	at forge.game.card.CardFactory.getCard(CardFactory.java:246)
> 	at forge.game.card.CardFactory.getCard(CardFactory.java:242)
> 	at forge.game.card.Card.fromPaperCard(Card.java:6532)
> 	at forge.game.card.Card.getCardForUi(Card.java:6543)
> 	at forge.game.card.CardView.getCardForUi(CardView.java:64)
> 	at forge.card.CardRenderer.drawCard(CardRenderer.java:606)
> 	at forge.itemmanager.views.ImageView$ItemInfo.draw(ImageView.java:991)
> 	at forge.itemmanager.views.ImageView$Pile.draw(ImageView.java:920)
> 	at forge.itemmanager.views.ImageView$Group.draw(ImageView.java:829)
> 	at forge.Graphics.draw(Graphics.java:176)
> 	at forge.toolbox.FContainer.draw(FContainer.java:64)
> 	at forge.toolbox.FScrollPane.draw(FScrollPane.java:332)
> 	at forge.Graphics.draw(Graphics.java:176)
> 	at forge.toolbox.FContainer.draw(FContainer.java:64)
> 	at forge.Graphics.draw(Graphics.java:176)
> 	at forge.toolbox.FContainer.draw(FContainer.java:64)
> 	at forge.Graphics.draw(Graphics.java:176)
> 	at forge.toolbox.FContainer.draw(FContainer.java:64)
> 	at forge.screens.FScreen.draw(FScreen.java:164)
> 	at forge.Forge.render(Forge.java:887)
> 	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Window.update(Lwjgl3Window.java:387)
> 	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.loop(Lwjgl3Application.java:192)
> 	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:166)
> 	at forge.app.Main.main(Main.java:148)
> Caused by: java.lang.RuntimeException: AbilityFactory:getAbility: crash when trying to create ability  of card: Myrkul's Edict
> 	at forge.game.ability.AbilityFactory.getAbility(AbilityFactory.java:157)
> 	at forge.game.ability.AbilityFactory.getAbility(AbilityFactory.java:116)
> 	at forge.game.ability.AbilityFactory.getAbility(AbilityFactory.java:113)
> 	at forge.game.card.CardFactoryUtil.addAbilityFactoryAbilities(CardFactoryUtil.java:595)
> 	... 26 more
> Caused by: java.lang.RuntimeException: AbilityFactory : getAbility -- no API in Myrkul's Edict: ChoosePlayer | Defined$ You | Choices$ Player.Opponent | SubAbility$ DBSac | StackDescription$ SpellDescription | SpellDescription$ 1—9 VERT Choose an opponent. That player sacrifices a creature.
> 	at forge.game.ability.AbilityFactory.getAbility(AbilityFactory.java:145)
> 	at forge.game.ability.AbilityFactory.getAbility(AbilityFactory.java:174)
> 	at forge.game.ability.AbilityFactory.getSubAbility(AbilityFactory.java:457)
> 	at forge.game.ability.AbilityFactory.getAbility(AbilityFactory.java:299)
> 	at forge.game.ability.AbilityFactory.getAbility(AbilityFactory.java:179)
> 	at forge.game.ability.AbilityFactory.getAbility(AbilityFactory.java:148)
> 	... 29 more
> ```

`SVar:OneOppSac:ChoosePlayer `   it misses `DB$ `

--- a/forge-gui/res/cardsfolder/upcoming/moonshae_pixie.txt
+++ b/forge-gui/res/cardsfolder/upcoming/moonshae_pixie.txt
@@ -4,7 +4,7 @@ Types:Creature Faerie
 PT:2/2
 K:Flying
 T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Self | Execute$ TrigDraw | TriggerDescription$ When CARDNAME enters the battlefield, draw cards equal to the number of opponents who were dealt combat damage this turn.
-SVar:TrigDraw:AB$ Draw | NumCards$ X
+SVar:TrigDraw:DB$ Draw | NumCards$ X
 SVar:X:PlayerCountRegisteredOpponents$HasPropertywasDealtCombatDamageThisTurn
 AlternateMode:Adventure
 Oracle:Flying\nWhen Moonshae Pixie enters the battlefield, draw cards equal to the number of opponents who were dealt combat damage this turn.

--- a/forge-gui/res/cardsfolder/upcoming/myrkuls_edict.txt
+++ b/forge-gui/res/cardsfolder/upcoming/myrkuls_edict.txt
@@ -2,7 +2,7 @@ Name:Myrkul's Edict
 ManaCost:1 B
 Types:Sorcery
 A:SP$ RollDice | Sides$ 20 | ResultSubAbilities$ 1-9:OneOppSac,10-19:EachOppSac,20:SacTopPower | SpellDescription$ Roll a d20.
-SVar:OneOppSac:ChoosePlayer | Defined$ You | Choices$ Player.Opponent | SubAbility$ DBSac | StackDescription$ SpellDescription | SpellDescription$ 1—9 VERT Choose an opponent. That player sacrifices a creature.
+SVar:OneOppSac:DB$ ChoosePlayer | Defined$ You | Choices$ Player.Opponent | SubAbility$ DBSac | StackDescription$ SpellDescription | SpellDescription$ 1—9 VERT Choose an opponent. That player sacrifices a creature.
 SVar:DBSac:DB$ Sacrifice | Defined$ Chosen | SacValid$ Creature | SubAbility$ DBCleanupChosen
 SVar:EachOppSac:DB$ Sacrifice | Defined$ Player.Opponent | SacValid$ Creature | StackDescription$ SpellDescription | SpellDescription$ 10—19 VERT Each opponent sacrifices a creature.
 SVar:SacTopPower:DB$ RepeatEach | RepeatPlayers$ Player.Opponent | RepeatSubAbility$ DBChooseCard | SubAbility$ DBSacAll | StackDescription$ SpellDescription | SpellDescription$ 20 VERT Each opponent sacrifices a creature with the greatest power among creatures that player controls.

